journalpump/senders/rsyslog.py

```diff
@@ -53,6 +59,27 @@ def _init_rsyslog_client(self):
             self.rsyslog_client = None
             time.sleep(5.0)
 
+    def send_heartbeat(self):
+        if not self.rsyslog_client:
+            self._init_rsyslog_client()
+
+        try:
+            self.rsyslog_client.log(
+                facility=self.config.get("heartbeat_facility", 5),
+                severity=self.config.get("heartbeat_severity", 7),
+                timestamp=datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"),
+                hostname=self.config.get("heartbeat_hostname", socket.gethostname()),
+                program=self.config.get("heartbeat_program", "journalpump"),
+                msg=self.config.get("heartbeat_message", "HEARTBEAT"),
+            )
+        except RSYSLOG_CONN_ERRORS as ex:
+            self.mark_disconnected(ex)
+            raise ex
```

Comment:
Perhaps this should log warning and return instead of raising exception, otherwise base sender will print full stack trace, which might be spammy.

Reply:
i can convert the logging in the base sender to warn too, wdyt?